optuna/_hypervolume/wfg.py

```diff
@@ -147,6 +147,8 @@ def compute_hypervolume(
     if not np.all(np.isfinite(reference_point)):
         # reference_point does not have nan, thanks to the verification above.
         return float("inf")
+    if loss_vals.size == 0:
+        return 0.0
```

Comment:
Could you add a unit test for this change?

Reply:
Thank you for the suggestion, I've added it:)